pmm-app/src/pmm-add-instance-app-panel/DiscoveryPanel/Search.tsx

```diff
@@ -0,0 +1,54 @@
+import { Form as FormFinal } from 'react-final-form';
+import React, { ReactElement } from 'react';
+import { useForm } from 'react-final-form-hooks';
+import { InputField } from '../../react-plugins-deps/components/FieldsComponents/Input/Input';
+import { PasswordField } from '../../react-plugins-deps/components/FieldsComponents/Password/Password';
+
+const Search = props => {
```

Comment:
What do we search here?
Maybe it's better to rename to `CredentialsForm` or something else?